pkg/kubelet/kubelet_pods.go

```diff
@@ -188,12 +191,32 @@ func makeMounts(pod *v1.Pod, podDir string, container *v1.Container, hostName, h
 			}
 		}
 
+		var propagation runtimeapi.MountPropagation
+		if enableMountPropagation {
```

Comment:
nit: Can this logic be in a separate method?

Reply:
extracted into a separate method